benchmarks/README.md

```diff
@@ -0,0 +1,54 @@
+# IREE Benchmarks
+
+This directory contains configuration definition for IREE's continuous
+benchmarks suite. Benchmark results are posted to https://perf.iree.dev.
+
+The https://buildkite.com/iree/iree-benchmark Buildkite pipeline runs on each
+commit to the `main` branch and posts those results to the dashboard. The
+pipeline also runs on pull requests with the `buildkite:benchmark` label,
+posting results compared against their base commit as comments.
+
+## Types of benchmarks
+
+```
+├── TensorFlow
+│     * models authored in TensorFlow and imported with `iree-import-tf`
+└── TFLite
+      * models converted to TensorFlow Lite and imported with `iree-import-tflite`
+```
+
+## Adding new benchmarks
+
+### Machine learning model latency
+
+1. Pick the model you want to benchmark and find its source, which could be
+   a Python script, TensorFlow SavedModel from https://tfhub.dev/, TensorFlow
+   Lite FlatBuffer, or some other format with a supported path into IREE. The
+   model can optionally include trained weights if those are important for
+   benchmarking.
+
+2. Import the model into an MLIR file that IREE can compile using the core
+   `iree-translate` tool. For TensorFlow models use `iree-import-tf`, for
+   TensorFlow Lite models use `iree-import-tflite`, etc. Take notes for where
+   the model came from and how it was imported in case the MLIR file needs to
+   be regenerated in the future.
```

Comment:
I'd put a TODO here to mention that eventually we'd like to avoid invoking `iree-import-tf`/`iree-import-tflite` separately. Instead we'd like to generate/host from the original source. Concretely, for TF we'd like to generate from Python model, for TFLite might still need to host the FlatBuffer.

Reply:
I added a note and linked to your gist + my Colab notebook as references

Reply:
https://github.com/google/iree/issues/6942

Reply:
Thanks. Added a link to that as well.